src/Apps/Legion/Services/StoryUtilities.tsx

```diff
@@ -0,0 +1,156 @@
+/**
+ * @summary This file contains helper functions for composing Story tests
+ */
```

Comment:
do we really need a duplicate for this?

Reply:
Removed duplicates, I'll bring them back if our plan is to lift and shift the App folder.